WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -0,0 +1,333 @@
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Linq;
+using System.Windows.Input;
+using Avalonia.Threading;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Fluent.State;
+using WalletWasabi.WabiSabi.Client;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets;
+
+public partial class CoinJoinStateViewModel : ViewModelBase
+{
+	enum State
+	{
+		Disabled,
```

Comment:
It's a common practice to add an **"Invalid"** or **"Default"** value (with integer value `0`) to detect possible bugs in the future.

Example: 

```csharp
enum State
{
     Invalid = 0,
     Disabled,
     AutoCoinJoin,
     ....
}
```

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1008

Reply:
done.